### PR TITLE
feat(ci): add dockerhub as repo target during image push

### PR DESCRIPTION
**Pull request by c3ppc3pp**

### CODE REVIEW

File: .github/workflows/full.yml [hunk 1]
```diff
@@ -459,9 +457,17 @@ jobs:
...
         if: github.ref_type == 'tag'
         run: |
-          docker tag ${REGISTRY}/${CRANE_IMAGE_NAME}:${{ needs.changes.outputs.version }}
-          docker tag ${REGISTRY}/${DAGENT_IMAGE_NAME}:${{ needs.changes.outputs.version }}
-          docker tag ${REGISTRY}/${CLI_IMAGE_NAME}:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${CRANE_IMAGE_NAME}:${{ needs.changes.outputs.fallbacktag }} ${REGISTRY}/${CRANE_IMAGE_NAME}:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${DAGENT_IMAGE_NAME}:${{ needs.changes.outputs.fallbacktag }} ${REGISTRY}/${DAGENT_IMAGE_NAME}:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${CLI_IMAGE_NAME}:${{ needs.changes.outputs.fallbacktag }} ${REGISTRY}/${CLI_IMAGE_NAME}:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${CRANE_IMAGE_NAME}:${{ needs.changes.outputs.fallbacktag }} docker.io/dyrectorio/crane:${{ needs.changes.outputs.version }}
+          docker tag ${REGISTRY}/${DAGENT_IMAGE_NAME}:${{ needs.changes.outputsfallbacktag }} docker.io/dyrectorio/dagent:${{ needs.changes.outputs.version }}
```
Comment (polaroi8d) [inline .github/workflows/full.yml:464]: Could you use pre-defined variables for this `docker.io/dyrectorio`?

**Review by polaroi8d — COMMENTED** [review]
I don't know this is better, but we can tag multiple image in a single line like this: `docker build -t name1:tag1 -t name1:tag2 -t name2 .`

What do you think?

File: .github/workflows/full.yml (hunk 1 above)
Comment (c3ppc3pp) [reply .github/workflows/full.yml]: Was planned, hence the draft status.

**Comment by c3ppc3pp** [comment]
> I don't know this is better, but we can tag multiple image in a single line like this: `docker build -t name1:tag1 -t name1:tag2 -t name2 .`
> 
> What do you think?

You are right. Fixed.

**Review by nandor-magyar — APPROVED** [review]

**Review by polaroi8d — APPROVED** [review]

**Pull request merged**